08_lisp.md

```diff
@@ -240,10 +268,11 @@ Return the next, otherwise the user will have to save it before freeing.
       return cdr;
     }
 
-This is the same as `(setf (cdr x) free-list)` in Lisp or `(set-cdr! x free-list)`
-in Scheme.
+[^free-in-lisp]
```

Comment:
This footnote looks out of place. I think it was supposed to be attached to the previous paragraph before the code block.